crates/ruff_python_ast/ast.toml

```diff
@@ -66,32 +66,237 @@ add_suffix_to_is_methods = true
 anynode_is_label = "statement"
 doc = "See also [stmt](https://docs.python.org/3/library/ast.html#ast.stmt)"
 
-[Stmt.nodes]
-StmtFunctionDef = {}
-StmtClassDef = {}
-StmtReturn = {}
-StmtDelete = {}
-StmtTypeAlias = {}
-StmtAssign = {}
-StmtAugAssign = {}
-StmtAnnAssign = {}
-StmtFor = {}
-StmtWhile = {}
-StmtIf = {}
-StmtWith = {}
-StmtMatch = {}
-StmtRaise = {}
-StmtTry = {}
-StmtAssert = {}
-StmtImport = {}
-StmtImportFrom = {}
-StmtGlobal = {}
-StmtNonlocal = {}
-StmtExpr = {}
-StmtPass = {}
-StmtBreak = {}
-StmtContinue = {}
-StmtIpyEscapeCommand = {}
+[Stmt.nodes.StmtFunctionDef]
+doc = """See also [FunctionDef](https://docs.python.org/3/library/ast.html#ast.FunctionDef)
+and [AsyncFunctionDef](https://docs.python.org/3/library/ast.html#ast.AsyncFunctionDef).
+
+This type differs from the original Python AST, as it collapses the synchronous and asynchronous variants into a single type."""
+fields = [
+  { name = "is_async", type = "bool" },
+  { name = "decorator_list", type = "Decorator*" },
+  { name = "name", type = "Identifier" },
+  # check remove
+  { name = "type_params", type = "Box<crate::TypeParams>?" },
```

Comment:
This was boxed in the source code. But it's not Boxed in `StmtTypeAlias`.

Reply:
You can try, but I suspect that `StmtFunctionDef` is simply the largest statement node and reducing the size of any other node doesn't help to reduce the size of `Stmt`.